src/app/interfaces/category.ts

```diff
@@ -0,0 +1,4 @@
+export interface Category {
```

Comment:
good approach - to name interfaces with I in the beginning
ICategory, ICatRequestProperties, IContentItem etc.